src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/Common/TensorPrimitives.IUnaryOneToFourOperator.cs

```diff
@@ -0,0 +1,164 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics;
+
+#pragma warning disable CS8500 // This takes the address of, gets the size of, or declares a pointer to a managed type
+
+namespace System.Numerics.Tensors
+{
+    public static unsafe partial class TensorPrimitives
+    {
+        /// <summary>Operator that takes one input value and returns a single value.</summary>
+        /// <remarks>The input type must be half the size of the output type.</remarks>
```

Comment:
Maybe my math is wrong, but shouldn't this be:
```suggestion
        /// <remarks>The input type must be a quarter of the size of the output type.</remarks>
```